packages/dart/sshnoports/lib/src/noports_cli/activate/np_activate_params.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:sshnoports/src/noports_cli/util/constants.dart';
+import 'package:sshnoports/src/noports_cli/util/regex.dart';
```

Comment:
A params class that holds all currently necessary parameters and a fromArgs() method